modules/uhttp/filters.go

```diff
@@ -83,13 +81,12 @@ func (f tracingServerFilter) Apply(ctx context.Context, w http.ResponseWriter, r
 
 // authorizationFilter authorizes services based on configuration
 type authorizationFilter struct {
-	authClient  auth.Client
-	authCounter tally.Counter
+	authClient auth.Client
 }
 
 func (f authorizationFilter) Apply(ctx context.Context, w http.ResponseWriter, r *http.Request, next Handler) {
 	if err := f.authClient.Authorize(ctx); err != nil {
-		f.authCounter.Inc(1)
+		stats.HTTPAuthFailCounter.Inc(1)
```

Comment:
It'd be nice if the error returned from any authn/authz checks included tags for this counter: who's the caller, what scope of permissions did they fail to get, etc.